Game.cpp

```diff
@@ -255,9 +260,76 @@ void Game::print(ostream &stream) const
 	// Bottom border
 	stream << icons.at(Game::Icon::BOTTOM_LEFT);
 
-	for (int x = 0; x < map->getWidth(); x++)
+	for (int x = bottomLeft.x; x <= bottomRight.x; x++)
 	{
 		stream << icons.at(Game::Icon::HORIZONTAL);
 	}
 	stream << icons.at(Game::Icon::BOTTOM_RIGHT) << endl;
+}
+
+Location Game::getCorner(CornerType ct, bool mapCorner) const
+{
+	int x = 0, y = 0;
+	if (!mapCorner && this->hero == nullptr)
```

Comment:
A függvény meghívásakor a `mapCorner = this->hero == nullptr`, ez így soha nem következik be. Hozzátéve a játék indítása csak hero betöltés után lehetséges (`Game::run()` -> `throw Game::NotInitializedException();`)

Biztos szükséges (ilyen formában)?

Reply:
Igen, mert print viszont lehetséges. Azért raktam bele, hogy birja a unit testeket.

Reply:
Ha a unit tesztekhez szükséges is, szükséges a paraméter, vagy elég helyi változóban ellenőrizni?

Reply:
mapCorner-re gondolsz?
Azért paraméterként adom át, jelengleg még feleslegesen, hogy késöbb az Observer-nál nekelljen ezzel játszani, de kivehetem, ha úgy jobb.

Reply:
Ok. Akkor maradhat.
_Másik fájlba fog kerülni, más felépítéssel, mindenképp módosítani kell majd rajta (valamennyit)_